implementations/rust/ockam/ockam_node/src/context.rs

```diff
@@ -156,16 +166,40 @@ impl Context {
         tokio::time::sleep(dur).await;
     }
 
-    /// Create a new detached context without spawning a full worker
+    /// Create a new detached `Context` that will apply the given
+    /// [`AccessControl`] to any incoming messages it receives
+    pub async fn new_repeater<AC>(&self, access_control: AC) -> Result<RepeaterContext>
```

Comment:
I think it's the same as DetachedContext, we just need function that would allow passing AccessControl

Reply:
Yup, It's also just a type alias to `Context` for now but I'd like to start typing them according to function so long in anticipation of future refactors.